ipython_parallel/_version.py

```diff
@@ -1,2 +1,9 @@
-version_info = (4, 0, 0, 'dev')
+_version_major = 4
+_version_minor = 0
+_version_patch = 0
+_version_extra = 'dev'
```

Comment:
Can you revert the changes in _version_.py, please and leave it how it was? It was like that for a reason.